### DISCUSSION TITLE
Any way to click a fab?

### DISCUSSION THREAD

**Discussion by ColtonIdle** [mobile-dev-inc/Maestro#130] [Q&A] [answered]
I'm trying to run the sample from the readme

# flow_contacts_android.yaml

appId: com.android.contacts
---
- launchApp
- tapOn: "Create new contact"
- tapOn: "First Name"
- inputText: "John"
- tapOn: "Last Name"
- inputText: "Snow"
- tapOn: "Save"

but on the emulator there is no "create new contact", but there is a FAB with a + sign. I'm trying to figure out a way to click it. Is there some way to tell it to just click a FAB or something? I'm new to maestro, but yeah. how would i click that?

**Comment by k0shk0sh** [comment, marked as answer]
You can use the fab id: 

```
- tapOn: 
     id: com.twitter.android:id/fab_id
```
